dash-pipeline/dockerfiles/Dockerfile.p4c-bmv2

```diff
@@ -24,23 +24,23 @@ FROM amd64/ubuntu:20.04
 # Need python for "p4c" wrapper &  gcc for the C preprocessor
 RUN apt update && apt install -y python3 gcc
 
-COPY --from=p4lang-p4c /usr/local/lib/* /usr/local/lib/
+COPY --from=chrissommers/dash-p4c-bldr:221007 /usr/local/lib/* /usr/local/lib/
```

Comment:
Would you consider leaving existing recipes unchanged and come up with a new image name e.g. dash-p4c-XXX-bldr where XXX somehow indicates the custom or stateful nature of this, maybe even "pna"?